### DISCUSSION TITLE
Drag-to-rearrange not working

### DISCUSSION THREAD

**Discussion by scrutinizer11** [ViennaRSS/vienna-rss#1831] [Q&A]
**Describe the bug**
See the title.

**To Reproduce**
Drag any subscription to change the order or add to a folder.

**Please complete the following information:**
 - Vienna version 3.9.3
 - OS version: 10.14

**Additional information:**
In which version of Vienna does the problem not occur, if applicable:

Vienna 2.5 on macOS 10.9

**Comment by barijaona** [comment]
Please go to the "View" -> "Order By" menu and make sure that "Manual" is selected.
